README.md

```diff
@@ -501,20 +503,19 @@ Call the provided function, `fn`, with up to `n` arguments, using `Array.slice(0
 
 ```js
 const ary = (fn, n) => (...args) => fn(...args.slice(0, n));
-```
+```  
 
 <details>
-<summary>Examples</summary>
-
-```js
+  <summary>Examples</summary>
+  
+  ```js
 const firstTwoMax = ary(Math.max, 2);
 [[2, 6, 'a'], [8, 4, 6], [10]].map(x => firstTwoMax(...x)); // [6, 8, 10]
 ```
 
 </details>
-
-<br>[⬆ Back to top](#table-of-contents)
-
+  
+[⬆ Back to top](#table-of-contents)
```

Comment:
The `<br>` was there for a reason: to add soime extra space that Markdown didn't want to add with a line break.

Reply:
right!